packages/common/src/types.ts

```diff
@@ -16,11 +16,18 @@ export interface Chain {
   name: string
   chainId: number
   networkId: number
+  // TODO: make mandatory in next breaking release
+  defaultHardfork?: string
```

Comment:
I was about to suggest this to indeed make `defaultHardfork` mandatory.

Reply:
If someone is using this type this could otherwise break things unfortunately.

Reply:
Yep but we can do this in the next major release (maybe add this to the V6 tracking issue?)